modules/administration/pages/ansible-integration.adoc

```diff
@@ -31,11 +32,11 @@ Supported features are:
 == Requirements
 
 To use Ansible features, you need to register your Ansible Control Node system as a Salt client to your {productname} Server.
-When registered, you must enable the ``Ansible Control Node`` system type in the [guimenu]``Add-on System Types`` on the menu:System Details[Properties] page of that system.
+When registered, you must enable the ``Ansible Control Node`` system type in the [guimenu]``Add-on System Types`` on the [guimenu]``System Details > Properties`` page of that system.
 
-Enabling this system type ensures that the ``ansible`` package is installed on the system by adding it in the highstate and activates the Ansible features under the menu:System Details[Ansible] page.
+Enabling this system type ensures that the ``ansible`` package is installed on the system by adding it in the highstate and activates the Ansible features under the [guimenu]``System Details > Ansible`` page.
 
-As the next step, configure the paths to your Ansible inventories and playbooks on the menu:System Details[Ansible > Control Node] page.
+As the next step, configure the paths to your Ansible inventories and playbooks on the [guimenu]``System Details > Ansible > Control Node`` page.
```

Comment:
I think can and should stay with "menu:".  Or is there a reason why you changed it?

Reply:
menu is not rendered properly in the HTML, and shows as plain text.

Reply:
But we use it a lot in modules/reference.  @jcayouette did something change with the adoc parser or renderer?

Reply:
I will take a look. Perhaps we need a css rule.